facebook_scraper/extractors.py

```diff
@@ -203,6 +206,16 @@ def extract_time(self) -> PartialPost:
             except (KeyError, ValueError):
                 continue
 
+        try:
+            time_match = self.time_regex.search(self.element.full_text)
+            if time_match:
+                time = time_match.group(0)
+                return {
+                    'time': dateparser.parse(time)
```

Comment:
https://github.com/kevinzg/facebook-scraper/pull/135#pullrequestreview-524168255

Reply:
I'm not sure I understood the problem, you tried to scrap a post with the date 'October 13 at 9:44 PM', or did you send the string 'at 13' to dateparser?

I would love seeing the original post if so.

Reply:
The thing is that the matched group was `ASFOROctober 13 at 8:44 PM`, including a part of the page name or title.
I came with this ugly regex fix
`time_regex = re.compile(r"((Jan(?:uary)?|Feb(?:ruary)?|Mar(?:ch)?|Apr(?:il)?|May|Jun(?:e)?|Jul(?:y)?|Aug(?:ust)?|Sep(?:tember)?|Oct(?:ober)?|Nov(?:ember)?|Dec(?:ember)?) \d{1,2} at \d{1,2}:\d{2} (AM|PM))|(\d{1,2} \w+)")` 

and that's how `(\d{1,2} \w+)` ended up finding `13 at`

Reply:
Added also regex for `Yesterday at 12:30 PM`-like matching also:
`((Jan(?:uary)?|Feb(?:ruary)?|Mar(?:ch)?|Apr(?:il)?|May|Jun(?:e)?|Jul(?:y)?|Aug(?:ust)?|Sep(?:tember)?|Oct(?:ober)?|Nov(?:ember)?|Dec(?:ember)?) \d{1,2} at \d{1,2}:\d{2} (AM|PM))|Yesterday at \d{1,2}:\d{2}|(\d{1,2} \w+)`

Reply:
Thks balazssandor, there is also has Today, so will be this:
((Jan(?:uary)?|Feb(?:ruary)?|Mar(?:ch)?|Apr(?:il)?|May|Jun(?:e)?|Jul(?:y)?|Aug(?:ust)?|Sep(?:tember)?|Oct(?:ober)?|Nov(?:ember)?|Dec(?:ember)?) \d{1,2} at \d{1,2}:\d{2} (AM|PM))|Today at \d{1,2}:\d{2} (AM|PM)|Yesterday at \d{1,2}:\d{2} (AM|PM)|(\d{1,2} \w+)